<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FIOMqtt</class>
 <widget class="QWidget" name="FIOMqtt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="DataButton" name="pushButton_in3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_relay3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DataButton" name="pushButton_in2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_server">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>broker.emqx.io</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_server">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>服务器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_download">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>571</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>FourIO/DownLoad/43151025325847310673FF30</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_server">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_upload">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>571</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>FourIO/UPLoad/43151025325847310673FF30</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_upload">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>上行主题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_download">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>下行主题</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataButton</class>
   <extends>QPushButton</extends>
   <header>databutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
